pkg/server/login/locales.go

```diff
@@ -0,0 +1,56 @@
+package login
+
+import "golang.org/x/text/language"
+
+func getLocale(langBase string) map[string]string {
+	switch langBase {
+	case language.English.String():
+		return locale_en
+	// Uncomment once given locale is available
```

Comment:
When is that going to be?

Reply:
Thats up to translations team that is providing those.
@sg00dwin any estimations ?

Reply:
Full-time translators typically use specialized file formats/tools/editors, a repository of previously-translated strings, and the like. How is this going to interoperate with such workflows?

Is this really such a unique and special case that asking translators to provide Go patches, or setting up a special process to manually turn contributed translations into Go patches, is worthwhile and practical?

Reply:
@mtrmac We have these tools in place for the main console repo. We thought that would be overkill here for a handful of messages that rarely change. We were planning to include whatever strings we need with the other console strings, and the console team would manage patching them when needed.

Reply:
> Thats up to translations team that is providing those.
> @sg00dwin any estimations ?

The translation team doesn't provide estimated turn around times. Still waiting.

Reply:
Translated strings are now included for each language.